packages/next/src/server/base-server.ts

```diff
@@ -1098,6 +1099,22 @@ export default abstract class Server<
             addRequestMeta(req, 'postponed', postponed)
           }
 
+          // If the request is a next data request and it has a postponed state,
+          // we should error, as it represents an unprocessable request.
+          if (
+            getRequestMeta(req, 'isNextDataReq') &&
+            getRequestMeta(req, 'postponed')
+          ) {
+            // The server understood that this is a PPR resume request, as the
+            // headers were included to correctly indicate a resume request, but
+            // because the request URL indicates that this should render a next
+            // data route (a pages router route), this represents an
+            // unprocessable request.
+            res.statusCode = 422
+            res.send()
```

Comment:
double checking, but other errors seem to call `res.renderError` when we have a non-200 status code. Should we do the same here?